src/components/dictionaryConcepts/containers/CreateConcept.jsx

```diff
@@ -177,7 +177,8 @@ export class CreateConcept extends Component {
   addDataFromDescription(data) {
     const currentData = this.state.descriptions.filter(description => description.id === data.id);
     if (currentData.length) {
-      const newList = this.state.descriptions.map(description => (description.id === data.id ? data : description));
+      const newList = this.state.descriptions.map(description =>
+        (description.id === data.id ? data : description));
```

Comment:
Adjust that spacing between the two lines above

Reply:
@alexmochu The reason was because it exceeded the maximum length of a line. The update has been made.